bin/gen_ide_helper_models.php

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
```

Comment:
It would be nicer if the file has no `.php` extension ;)

Reply:
✓ done